pkg/vsphere/tasks/waiter.go

```diff
@@ -152,6 +159,11 @@ func isRetryError(op trace.Operation, err error) bool {
 		if f, ok := err.(types.HasFault); ok {
 			logFault(op, f.Fault())
 		} else {
+			// retry the temporary errors
```

Comment:
There is a DoNotRetry error used in `pkg/fetcher` and I'm sure similar logic in `pkg/retry`. I have a todo item to get retry and tasks package working together or unified. This is just informational.